scripts/import_from_ena.py

```diff
@@ -27,22 +33,57 @@
 DATA_SOURCES = ['fastq', 'sra', 'cram_index']
 DATA_TYPES = ['ftp', 'galaxy', 'aspera']
 
+logger = logging.getLogger(__name__)
+logging.basicConfig(format='%(asctime)s\t%(levelname)s:\t%(name)s line %(lineno)s\t%(message)s', level=logging.INFO)
+# suppress logging information from elasticsearch
+logging.getLogger('elasticsearch').setLevel(logging.WARNING)
 
-def main():
+
+@click.command()
+@click.option(
+    '--es_hosts',
+    default="wp-np3-e2;wp-np3-e3",
+    help='Specify the Elastic Search server(s) (port could be included), e.g. wp-np3-e2:9200. '
+         'If multiple servers are provided, please use ";" to separate them, e.g. "wp-np3-e2;wp-np3-e3"'
+)
+@click.option(
+    '--es_index_prefix',
+    default="",
+    help='Specify the Elastic Search index prefix, e.g. '
+         'faang_build_1_ then the indices will be faang_build_1_experiment etc.'
+         'If not provided, then work on the aliases, e.g. experiment'
+)
+# TODO check single or double quotes
+def main(es_hosts, es_index_prefix):
     """
     Main function that will import data from ena
+    :param es_hosts:
```

Comment:
Need a documentation for these parameters

Reply:
done in 9d53f80